### ISSUE TITLE
bimvie.ws dependent BIMserver ?

### ISSUE THREAD

**Issue by lvxiaomu** [opensourceBIM/bimvie.ws#60]
Hi ,

bimvie.ws and BIMserver can be installed on different machines ?

**Comment by rubendel** [comment]
Yes, just make sure you change the appropriate settings in settings.js (in BIMvie.ws):
- getBimServerApiAddress, should return the location of the BimServerJavaScriptApi (usually installed on a BIMserver)
- getBimSurferApiAddress should return the location of BIMsurfer (usually also installed on a BIMserver, but you can host your own, possibly modified, version)
- getStaticServerAddress, should call the callback function with the address on which the BIMserver API will be available. This is async because some implementation get this address by using I/O

**Comment by lvxiaomu** [comment]
I have another problem ，
when access to the page will be loaded `bimvie.ws-bimviews-0.0.57/plugin.version`,
but there is no `plugin.version` in the current directory

**Comment by lvxiaomu** [comment]
getVersion, just add a version number for the loaded file

`myRequest.open("GET", Global.baseDir + "plugin.version", true);`

This line of code will launch the following request 

```
Request URL:http://localhost:8080/bimvie.ws-bimviews-0.0.57/plugin.version
Request Method:GET
Status Code:404 Not Found
Remote Address:127.0.0.1:8080
```
I can't find the plugin.version file in the local path.

**Comment by rubendel** [comment]
That file does not exist, that's why you have to replace the code in the method with either:
```js
successCallback("123");
```
OR
```js
successCallback((new Date().getTime()));
```

**Comment by lvxiaomu** [comment]
![image](https://cloud.githubusercontent.com/assets/13172372/18777192/6c930ad2-81a0-11e6-8298-244535e5ce3f.png)

Now console no mistake, but the BIMviews is not loaded

**Comment by lvxiaomu** [comment]
![image](https://cloud.githubusercontent.com/assets/13172372/18777869/0a5d5e5e-81a4-11e6-97fc-ab9053558c0f.png)

![image](https://cloud.githubusercontent.com/assets/13172372/18778107/5d1c244e-81a5-11e6-905a-45277e251a76.png)


login method requires three parameters, How to get parameters `address` ?

**Comment by lvxiaomu** [comment]
![image](https://cloud.githubusercontent.com/assets/13172372/18780250/3049d4fc-81af-11e6-99ec-219426144c21.png)

```
<script>
  // https://github.com/rgrove/lazyload
  function l(b) {
    var a = m[b], c, f;
       if (a)
         c = a.callback,
         f = a.urls,
         f.shift(),
         h = 0,
         f.length || (c && c.call(a.context, a.obj),
         m[b] = null ,
         n[b].length && j(b))
  }
.....
</script>
```
the last time, in this part of the code  `f.length || (c && c.call(a.context, a.obj),`  c.call() is undefined, so it will be wrong
